docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+- Now configure MetaMask following the instructions [here](https://docs.obscu.ro/wallet-extension/configure-metamask.html)
+- Finally, check you can open the Remix IDE by visiting the following URL in your browser https://remix.ethereum.org
+
+That's it. You're all set to start building your first dApp on Obscuro.
+
+## 1. To begin, we'll clone a simple repo from Github through Remix by following these instructions
+
+#### Click on GitHub
+![Step 1 screenshot](https://images.tango.us/workflows/57863807-6b96-4295-a7a1-42da1623f030/steps/919b227f-f1f5-4af3-aba1-7fb83576d505/8144f3f5-7855-4c83-86dd-c88e5476c367.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=874%3A594)
+
+
+#### Paste "https://github.com/obscuronet/sample-applications/blob/main/number-guessing-game/contracts/Guess.sol" into input
+![Step 2 screenshot](https://images.tango.us/workflows/57863807-6b96-4295-a7a1-42da1623f030/steps/4b3b212d-2518-4cc9-aab2-4befe128b38c/66a9a87d-0899-42d2-8f6a-98f204b25825.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=530%3A371)
+
+
+#### Select Guess.sol, and we're ready
+![Step 1 screenshot](https://images.tango.us/workflows/bc1e6526-eb4e-4815-8690-412c37c1c5cb/steps/3a33b699-a3a9-42b5-b33a-2826f4516b08/faa2c399-1f01-4263-90d4-5d31f075e9bd.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=591%3A175)
+
+
+# Let's code
+Guess.sol is a Solidity file, and once we're done, it will contain everything we need for the Guessing Game. Inside the file you'll find the classic ERC-20 interface and a simple implementation named ERC20Basic representing OGG (Obscuro Guessing Game token). We'll use OGG as means of players entering to play and the prize pool.
+
+There is also a skeleton contract called Guess inside Guess.sol, and this is what we'll be extending. The contract Guess will be written the same as any other Solidity contract in Ethereum. It'll run on the EVM and will exhibit the same behaviours.
+
+![Step 1 screenshot](https://images.tango.us/workflows/41b454a9-b560-4d60-8775-3e40c5bc1466/steps/74347d9b-e3fe-48aa-8e74-fecebd22fdfa/80231e65-e0be-4bf2-8150-521721745b2c.png?crop=focalpoint&fit=crop&fp-x=0.3145&fp-y=0.5909&fp-z=3.3270&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=2307%3A1244)
+
+Let's start by defining the key variables we'll need for the game. One is to store the secret number that only the contract knows about, and another containing the ERC-20 contract.
```

Comment:
It wasn't too clear to me what these variables meant. I wonder if we rename `_target` to `secretNumber` or similar?

And I think you should change "containing the ERC-20 contract" to "containing the ERC-20 contract to manage OGG tokens".